### DISCUSSION TITLE
How to implement a hybrid inverse PINN.

### DISCUSSION THREAD

**Discussion by leocastel** [lululxvi/deepxde#1667] [Q&A]
Dear all,

I would like to write a code using DEEPXDE to find the coefficients **c1 and c2** of the differential equation:
**df(t)/dt=g(t,c1,c2)**.

But the training data is related to another function **I(t)**, which is related to **f(t)** by the following:
**I(t)=[a+bf(t)]V(t)**. In some points $V(t)=0$, which does not give the possibility of the trivial inversion of the equation.
I also would like to find the constants **a and b** through inverse-PINN.

1) Can I implement this problem?
2) Can I implement this problem when a and b are known?

Best regards,

**Comment by lululxvi** [comment]
Yes.
